src/main/java/parser/Parser.java

```diff
@@ -0,0 +1,134 @@
+package parser;
+
+import storage.Storage;
+import tasklist.TaskList;
+import ui.Ui;
+import mollyexception.MollyException;
+import task.*;
```

Comment:
As others have mentioned, wildcard imports is in violation of the coding standard in this project. You can configure IntelliJ so it does not auto contract the import statements.

https://se-education.org/guides/tutorials/intellijCodeStyle.html